### PR TITLE
support llama

### PR DESCRIPTION
**Pull request by pathfinder-pf**
support llama-3.1-8b-instruct etc

### CODE REVIEW

**Comment by pathfinder-pf** [comment]
Llama-3.1-8B-Instruct model reproduction
- server launch command
JAX_COMPILATION_CACHE_DIR=/tmp/jit_cache  python3 -u -m sgl_jax.launch_server --model-path meta-llama/Llama-3.1-8B-Instruct --trust-remote-code  --dist-init-addr=0.0.0.0:10011 --nnodes=1  --tp-size=4 --device=tpu --random-seed=3 --node-rank=0 --mem-fraction-static=0.8 --chunked-prefill-size=8192 --download-dir=/tmp --dtype=bfloat16 --precompile-bs-paddings 1 64 --max-running-requests 64 --max-total-tokens 257536 --skip-server-warmup --attention-backend=fa --precompile-token-paddings 8192 --page-size=64
- client(lm_eval 0.4.9.1)
lm_eval   --model local-chat-completions   --model_args model=meta-llama/Llama-3.1-8B-Instruct,base_url=http://localhost:30000/v1/chat/completions,num_concurrent=64 --tasks gsm8k_cot  --apply_chat_template --fewshot_as_multiturn
- compare
official vs sglang-jax
image
image
